lib/template.js

```diff
@@ -387,6 +404,20 @@ class Template {
                         }
                     });
                 }
+
+                if (propDef.dataFile) {
+                    if (!propDef.format) {
+                        propDef.format = 'hidden';
+                    }
+
+                    const dataFile = dataFiles[propDef.dataFile];
+                    if (propDef.base64) {
```

Comment:
Would it be worth making this two way instead with two keywords?
 
`fromBase64` (which will decode before using the value) 
and `toBase64` (which is the current conversion)

Reply:
I have added the two new keywords as suggested.